src/Instances/onDestroy.lua

```diff
@@ -40,7 +40,7 @@ local function onDestroy(instanceRef: PubTypes.SemiWeakRef, callback: (...any) -
 		end
 	end
 
-	local args = table.pack(...)
+	local args = {...}
```

Comment:
PRs should focus on a single change. Including multiple leads to messy commits and is generally not a good idea.

Is this even a good change? I feel like table.pack will be faster or at least open to optimisation compared to `{...}`.

Reply:
I came back to benchmark these in more detail, turns out I was wrong and `table.pack` *was* faster. Not sure how I managed to mess up the first set of results, sorry about that.

Reply:
In the future, you can do `git reset HEAD~1` to undo the last commit. The files will then still be modified, and you can change them, or undo the modifications altogether like here. Then, once you've done that, you'll need to _force-_push, with `git push -f`, to update the GitHub branch.

Reply:
Great, thanks, I'll do that next time.